cmake/protoc.cmake

```diff
@@ -4,4 +4,8 @@ set(protoc_files
 
 add_executable(protoc ${protoc_files})
 target_link_libraries(protoc libprotobuf libprotoc)
+
+# workaround for https://github.com/ClickHouse/ClickHouse/issues/7114
+target_link_libraries(protoc pthread)
```

Comment:
@proller (an expert in various build systems) recommends to search for Threads::Threads package.
But this variant is also acceptable.

Reply:
I've tried, but it didn't work. :/